### PR TITLE
Move crd checks from serving in to test

### PR DESCRIPTION
**Pull request by greghaynes**
These are generally usable test cases for K8s and as such should live in
pkg/test

### CODE REVIEW

**Comment by srinivashegde86** [comment]
thanks for doing this.

/lgtm
/approve

File: test/kube_checks.go [hunk 1]
```diff
@@ -73,3 +74,15 @@ func WaitForPodListState(client *KubeClient, inState func(p *corev1.PodList) (bo
 		return inState(p)
 	})
 }
+
+// GetConfigMap gets the knative serving config map.
+func GetConfigMap(client *KubeClient) k8styped.ConfigMapInterface {
+	return client.Kube.CoreV1().ConfigMaps("knative-serving")
```
Comment (dprotaso) [inline test/kube_checks.go]: Is there a reason why is the namespace hardcoded?

And does this helper apply to anything beyond `knative-serving`?

**Comment by dprotaso** [comment]
Also - https://github.com/knative/serving/issues/2703

File: test/kube_checks.go (hunk 1 above)
Comment (greghaynes) [reply test/kube_checks.go]: good point: https://github.com/knative/pkg/pull/237

**Pull request merged**